src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -173,25 +182,88 @@ private boolean isEligibleRefs(RefsChangedWebhookEvent event) {
         return true;
     }
 
+    private void getJobs(RefChangedDetails refChangedDetails, BitbucketWebhookTriggerRequest.Builder requestBuilder) {
```

Comment:
a `getJobs` returns `void` that's deceptive! :slightly_smiling_face:  Maybe we can rename this to something like `processJobs` instead?

Reply:
I think we missed this one when we renamed